### DISCUSSION TITLE
‎styles.css

### DISCUSSION THREAD

**Discussion by gabypds** [community/community#141926] [Programming Help]
### Select Topic Area

Question

### Body

Número da linha do arquivo original	Número da linha de diferença	Mudança de linha diferencial
@@ -3,6 +3,7 @@ body {
    background: black;
    margin: 0px;
    font-family: "Chakra Petch", sans-serif;
    margin-bottom: 100px;
}

header {
@@ -32,10 +33,6 @@ p {
    font-size: 20px;
}

img {
    height: 200px;
}
.categoria-videos {
    display: flex;
    overflow-x: auto;
@@ -45,4 +42,19 @@ img {
.categoria {
    padding-left: 20px;
    padding-right: 20px;
    margin-top: 50px;
}
.categoria-videos img {
    opacity: 0.5;
    height: 200px;
}
.categoria-videos img:hover {
    opacity: 1.0;
    border: 3px solid green;
}
.categoria h2 {
    color: rgb(42, 122, 228);
}

**Comment by fjyeo** [comment]
Not sure what your question is but It looks like you've added a `margin-bottom` to the body, updated the `.categoria-videos img` hover effect with a green border and opacity change, and adjusted some padding and color for `.categoria` elements. Let me know if you need help with any specific part of the changes you made though.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @gabypds ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
